lib/node-repo.js

```diff
@@ -137,15 +137,50 @@ function fetchLabelPages (options, startPageNum, cb) {
   })
 }
 
+function getBotPrLabels (options, cb) {
+  githubClient.issues.getEvents({
+    owner: options.owner,
+    repo: options.repo,
+    page: 1,
+    per_page: 100, // we probably won't hit this
+    issue_number: options.prId
+  }, (err, events) => {
+    if (err) {
+      return cb(err)
+    }
+
+    const ourLabels = []
+
+    for (const event of events) {
+      if (event.event === 'unlabeled') {
+        const index = ourLabels.indexOf(event.label.name)
+        if (index === -1) continue
+
+        ourLabels.splice(index, 1)
+      } else if (event.event === 'unlabeled') {
```

Comment:
Not certain this code will be reached. This the exact same `if` check as present in [line #154](https://github.com/nodejs/github-bot/pull/104/files#diff-3f74e33a82dabff73070dddacecc4dadR154) isn't it?